client/src/sections/Login/Login.tsx

```diff
@@ -1,16 +1,13 @@
-import React from 'react';
-import {useHistory} from 'react-router-dom';
+import React, {useState, useCallback} from 'react';
 import {Button, Card, Heading, ButtonGroup} from '@shopify/polaris';
-import logo from 'icons/logo.jpg';
+import logo from 'icons/logo.png';
 
 import './Login.css';
+import LoginModal from './LoginModal/LoginModal';
 
 export default function Login() {
-  const history = useHistory();
-
-  const handleLogin = () => {
-    history.push('/home');
-  };
+  const [showModal, setShowModal] = useState(false);
+  const handleChange = useCallback(() => setShowModal(!showModal), [showModal]);
```

Comment:
`toggleShowModal`

Reply:
toggleshowmodal over setshowmodal?

Reply:
I mean the function `handleChange` -> `toggleShowModal`